.github/workflows/nightly-perf-tests.yml

```diff
@@ -0,0 +1,111 @@
+name: Nightly Perf Tests
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: "0 6 * * *" # runs at 6 AM UTC, 1 AM EST.
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ github.ref }}-${{ github.sha }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ github.ref }}
+
+      - name: Gradle build service
+        run: ./gradlew --build-cache :service:build -x test
+
+  jib:
```

Comment:
`jib` doesn't appear to be used by any downstream steps, except `notify-slack`. I don't think it's needed for this workflow.

Once `jib` is removed, I think `build` can be removed too. The `integration` subproject doesn't depend on `service` so building `service` shouldn't have any effect on running the perf tests.

Reply:
`jib` is needed for Trivy to run against the generated image

Reply:
OK. Although now we're running trivy on PRs, after PR merge, and nightly. Would it be possible / worthwhile to put the trivy scan in its own workflow, so the workflow code isn't duplicated in the nightly and and PR workflows?

Reply:
A separate Trivy action would have to build the code and containerize it, so the build section would be duplicated across all the actions. As a result I'm not sure if that's worth separating it out into another action, especially since we leverage the build-cache for the actions, whereas a separate Trivy action would build from scratch.